SPECS/dhcp/dhcp.spec

```diff
@@ -72,6 +72,8 @@ cat > %{buildroot}/etc/dhcp/dhclient.conf << "EOF"
 #
 # Basic dhclient.conf(5)
 
+option rfc3442-classless-static-routes code 121 = array of unsigned integer 8;
```

Comment:
Can you add more details in the PR's summary about why we want to make this change through fast-track? We limit the changes on this branch to critical changes, which cannot wait for a regular monthly release (fixes for criticial/high CVEs, severely broken configuration, other critical security issues, etc.). I'm not an expert on DHCP configuration and couldn't tell, what this change is exactly about and how critical it may or may not be.